### PR TITLE
NCD, Progress Report - infrastructure + Personal Details pane

### PR DESCRIPTION
**Pull request by anvmn**
#448
#466

### CODE REVIEW

**Comment by anvmn** [comment]
Selection_1093

**Comment by anvmn** [comment]
@AronNovak please review

File: client/src/elm/Backend/NCDEncounter/Utils.elm [hunk 1]
```diff
@@ -22,3 +24,31 @@ getNCDEncountersForParticipant db participantId =
         |> Maybe.andThen RemoteData.toMaybe
         |> Maybe.map Dict.toList
         |> Maybe.withDefault []
+
+
+progressReportInitiatorFromUrlFragmemt : String -> Maybe NCDProgressReportInitiator
+progressReportInitiatorFromUrlFragmemt s =
```
Comment (AronNovak) [inline client/src/elm/Backend/NCDEncounter/Utils.elm]:
```suggestion
progressReportInitiatorFromUrlFragment s =
```

File: client/src/elm/Backend/NCDEncounter/Utils.elm [hunk 2]
```diff
@@ -22,3 +24,31 @@ getNCDEncountersForParticipant db participantId =
...
+            |> toEntityUuid
+            |> InitiatorRecurrentEncounterPage
+            |> Just
+
+    else
+        Nothing
+
+
+progressReportInitiatorToUrlFragmemt : NCDProgressReportInitiator -> String
+progressReportInitiatorToUrlFragmemt initiator =
```
Comment (AronNovak) [inline client/src/elm/Backend/NCDEncounter/Utils.elm]:
```suggestion
progressReportInitiatorToUrlFragment initiator =
```

**Review by AronNovak — APPROVED** [review]

**Pull request merged**